assessments/projects/contour/self-assessment.md

```diff
@@ -0,0 +1,206 @@
+
+* Vulnerability Response Process
+  * To report a vulnerability, an email must be sent to the Contour private email address and contain details of the vulnerability. The email must contain basic identification, detailed steps on how to reproduce the vulnerability, a description of the effects of the vulnerability, how the vulnerability affects Contour usage, and a list of other projects that were used in conjunction to produce the vulnerability.
+  * When a vulnerability is reported, the Contour Security Team is responsible for responding. Emails will be addressed within 3 business days and include a detailed plan to investigate the issue and any potential workarounds to perform in the meantime.
+  * The Security Team will investigate the vulnerability. If it is not deemed to be a vulnerability, the Team will follow up with a detailed reason for rejection. Otherwise, the Security Team will work on a plan to communicate and identify mitigating steps to protect users. Additionally, a Common Vulnerability Scoring System (CVSS) will be created using the CVSS calculator.
+
+* Incident Response 
+  * Github Pending security check
+
+  * The Security Team will work on fixing the vulnerability and perform internal testing.
+  * The Security Team will provide early disclosure of the vulnerbility via email to those who joined the Contour Distributors mailing list. Distributors can initially plan for the vulnerability patch ahead of the fix, and later can test the fix and provide feedback to the Contour team.
+  * The Security Team will patch the vulnerability in the next patch or minor release, once the fix is confirmed. The Team will then backport a patch release into all earlier supported releases.
+
+## Appendix
+
+* Known Issues Over Time
+  * ExternalName Services can be used to gain access to Envoy's admin interface
+    * https://github.com/projectcontour/contour/security/advisories/GHSA-5ph6-qq5x-7jwc
+    * Severity: High (8.5/10)
+    * Affected versions: all versions <= 1.16, 1.17.0
+    * Patched versions: v1.14.2, 1.15.2, 1.16.1, 1.17.1, 1.18.0 and up
+    * Statistics
+      * Attack vector:        Network
+      * Attack complexity:    Low
+      * Privileges required:  Low
+      * User interaction:     None
+      * Scope:                Changed
+      * Confidentiality:      Low
+      * Integrity:            None
+      * Availability:         High
+
+  * Ingress data plane (Envoy deployment) is vulnerable to DOS
+    * https://github.com/projectcontour/contour/security/advisories/GHSA-mjp8-x484-pm3r
+    * Severity: High
+    * Affected versions: 1.2.*-1.6.*
+    * Patched versions: 1.7
+
+* [CII Best Practices](https://www.coreinfrastructure.org/programs/best-practices-program/)
+  * TBD after discussion with project maintainers
```

Comment:
Is there an update to this? Looks like the project already has a passing badge - https://www.bestpractices.dev/en/projects/4141

Reply:
Thanks for pointing this out! We've updated the document to include the badge.